pytest_cli.py

```diff
@@ -0,0 +1,41 @@
+from collections.abc import Iterable
+from os import pardir, path
+import subprocess
+import sys
+
+from usmqe.usmqeconfig import UsmConfig
+
+conf = UsmConfig()
+
+# This serves as predefined pytest plugin configuration.
+# There can be added new options required by pytest plugins.
+params = {
+    "ansible_playbook_directory": path.abspath(
+        path.join(path.dirname(__file__), pardir, "usmqe-setup")),
+    "ansible_playbook_inventory": conf.config["inventory_file"]
+    }
+
+
+# `ansible_playbook_inventory` option is preprocessed
+if params['ansible_playbook_inventory']:
+    if isinstance(params['ansible_playbook_inventory'], Iterable):
+        params['ansible_playbook_inventory'] = params[
+            'ansible_playbook_inventory'][0]
+    else:
+        params['ansible_playbook_inventory'] = params[
+            'ansible_playbook_inventory']
```

Comment:
This is fishy. String is `Iterable` as well, so the else branch is never executed. Moreover this else branch looks pointless anyway: `params[a] = params[a]`.